client/components/happychat/style.scss

```diff
@@ -63,7 +63,7 @@
 		display: block;
 		background: lighten( $gray, 30% );
 		bottom: 0;
-		z-index: 10;
+		z-index: 1020;
```

Comment:
Why 1020? This should also be in our z-index map. The warning you were seeing earlier meant that that value was never used in any css file.